taxi/views.py

```diff
@@ -34,6 +37,26 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.edit.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer_list")
+    template_name = "taxi/manufacturer_form.html"
```

Comment:
Code Efficiency: Similar to previous comments, you don't need to override `template_name` if it's the same as the default one. By default, Django generates it using the format `<app name>/<model name>_form.html`. In this case, it would be `taxi/manufacturer_form.html`.